### ISSUE TITLE
New Guide: Import Twitter to WordPress

### ISSUE THREAD

**Issue by jordesign** [WordPress/data-liberation#34]
It would be great to have a guide on how to do a one-time import of your tweets as posts (if that is possible).

**Comment by colorful-tones** [comment]
Would the assumption be that Twitter posts would be imported as WP posts?

**Comment by jordesign** [comment]
Yes I think so... This is something that is currently possible using  https://github.com/beaulebens/keyring-social-importers
